x-pack/plugins/cases/public/components/edit_connector/index.tsx

```diff
@@ -13,64 +16,34 @@ import {
   EuiFlexItem,
   EuiButton,
   EuiButtonEmpty,
-  EuiLoadingSpinner,
   EuiButtonIcon,
 } from '@elastic/eui';
-import styled from 'styled-components';
 import { isEmpty, noop } from 'lodash/fp';
 
 import type { FieldConfig } from '@kbn/es-ui-shared-plugin/static/forms/hook_form_lib';
 import { Form, UseField, useForm } from '@kbn/es-ui-shared-plugin/static/forms/hook_form_lib';
-import type { Case } from '../../../common/ui/types';
-import type { ActionConnector, ConnectorTypeFields } from '../../../common/api';
+import type { Case, CaseConnectors } from '../../../common/ui/types';
+import type { ActionConnector, CaseConnector, ConnectorTypeFields } from '../../../common/api';
 import { NONE_CONNECTOR_ID } from '../../../common/api';
 import { ConnectorSelector } from '../connector_selector/form';
 import { ConnectorFieldsForm } from '../connectors/fields_form';
-import type { CaseUserActions } from '../../containers/types';
 import { schema } from './schema';
-import { getConnectorFieldsFromUserActions } from './helpers';
 import * as i18n from './translations';
 import { getConnectorById, getConnectorsFormValidators } from '../utils';
 import { usePushToService } from '../use_push_to_service';
-import type { CaseServices } from '../../containers/use_get_case_user_actions';
 import { useApplicationCapabilities } from '../../common/lib/kibana';
-import { useCasesContext } from '../cases_context/use_cases_context';
+import { PushButton } from './push_button';
+import { PushCallouts } from './push_callouts';
+import { normalizeActionConnector, getNoneConnector } from '../configure_cases/utils';
 
 export interface EditConnectorProps {
   caseData: Case;
-  caseServices: CaseServices;
-  connectorName: string;
-  connectors: ActionConnector[];
-  hasDataToPush: boolean;
+  caseConnectors: CaseConnectors;
+  allAvailableConnectors: ActionConnector[];
   isLoading: boolean;
-  isValidConnector: boolean;
-  onSubmit: (
-    connectorId: string,
-    connectorFields: ConnectorTypeFields['fields'],
-    onError: () => void,
-    onSuccess: () => void
-  ) => void;
-  userActions: CaseUserActions[];
+  onSubmit: (connector: CaseConnector, onError: () => void, onSuccess: () => void) => void;
 }
 
-const MyFlexGroup = styled(EuiFlexGroup)`
```

Comment:
I moved the styling of the text inside `x-pack/plugins/cases/public/components/connectors/card.tsx`. The result is the same as before.